example-playbooks/restore_scylla_manager_backup/restore.yaml

```diff
@@ -72,34 +72,27 @@
         name: scylla-server
         state: stopped
 
-- name: Restore schema
-  hosts: all[0]
-  become: true
-  gather_facts: false
-  tasks:
-    - name: Remove other seeds
+    - name: Temporarily make the first node a seed
       lineinfile:
         path: /etc/scylla/scylla.yaml
         regexp: '^(\s+) - seeds:'
         line: |
-          \g<1> - seeds: {{ inventory_hostname }}
+          \g<1> - seeds: {{ groups.all[0] }}
         backrefs: yes
 
-    - name: Download system_schema
+    - name: Download data
       shell: |
-        scylla-manager-agent download-files -L {{ backup_location }} -n {{ host_id[inventory_hostname] }} -T {{ snapshot_tag }} -d /var/lib/scylla/data/ -K system_schema --silent
+        scylla-manager-agent download-files -L {{ backup_location }} -n {{ host_id[inventory_hostname] }} -T {{ snapshot_tag }} -d /var/lib/scylla/data/
       become_user: scylla
+      async: 604800 # a week 7*24*60*60 seconds
+      poll: 5
 
-    - name: Start Scylla service
+    - name: Start Scylla service in the seed
       service:
         name: scylla-server
         state: started
+      when: inventory_hostname == groups.all[0]
 
-- name: Restore data
-  hosts: all
-  become: true
-  gather_facts: false
-  tasks:
     - name: Start Scylla service
```

Comment:
You need to start non-seeders sequentially - not all together.

Reply:
There is one seed at this point `inventory_hostname == groups.all[0]`

Reply:
Correct. But non-seeder nodes can't bootstrap in parallel. Scylla supports only a single bootstrapping node at a time.

Reply:
In my testing they seem to start fine but sure I'd fix that.